grace/run.py

```diff
@@ -63,28 +65,31 @@ def run_grace(config_file: Union[str, os.PathLike]) -> None:
     def return_unchanged(image, graph):
         return image, graph
 
-    # Prepare the feature extractor:
-    if config.extractor_fn is not None:
-        # Feature extractor:
-        extractor_model = torch.load(config.extractor_fn)
-        feature_extractor = FeatureExtractor(
-            model=extractor_model,
-            augmentations=img_patch_augs,
-            normalize=config.normalize,
-            bbox_size=config.patch_size,
-            keep_patch_fraction=config.keep_patch_fraction,
-        )
-    else:
-        feature_extractor = return_unchanged
-
     # Condition the augmentations to train mode only:
     def transform(
         image: torch.Tensor, graph: dict, *, in_train_mode: bool = True
     ) -> Callable:
+        # Prepare the feature extractor:
```

Comment:
this is part of PR #277 that we reviewed yesterday, right?

Reply:
Indeed, except I put this code inside the transform function as this is the only place where `FeatureExtractor` needs to be called.